docs/database/database-metrics.md

```diff
@@ -239,6 +241,176 @@ and SHOULD be provided **at span creation time** (if provided at all):
 
 ## Experimental
 
+### Database Response
+
+The following metric instruments describe database query response.
+
+#### Metric: `db.client.response.row_count`
+
+This metric is [required][MetricRequired].
```

Comment:
I'm not sure this metric should be required, as it is difficult to capture for some database systems, e.g. capturing it for JDBC requires tracking iteration over the result set

Reply:
For Cosmos DB, this is a requirement, though it may be optional for other databases. However, in theory, this applies to all databases—both relational and non-relational. The number of records returned by a query is a key factor contributing to response latency.

Reply:
you can say it's recommended on the general doc and say it's required in the cosmos doc.

It's not possible to reliably collect it in all cases (specifically when using byte-code instrumentation when paging is involved), so it can't be required in general case.